src/fiatconnect/ReviewScreen.test.tsx

```diff
@@ -56,6 +104,9 @@ describe('ReviewScreen', () => {
   beforeEach(() => {
     store.dispatch = jest.fn()
   })
+  afterAll(() => {
+    jest.resetAllMocks()
```

Comment:
what's the purpose of this reset in afterAll?

Reply:
Oops, removed